src/contractor/contractor.cpp

```diff
@@ -307,7 +308,7 @@ int Contractor::Run()
 
     util::Log() << "Loading edge-expanded graph representation";
 
-    util::DeallocatingVector<extractor::EdgeBasedEdge> edge_based_edge_list;
+    std::vector<extractor::EdgeBasedEdge> edge_based_edge_list;
```

Comment:
Any particular reason why you switched from `DeallocatingVector` to `std::vector`? My last status on this was that `DeallocatingVector` saved some memory, but I never evaluated if this is still true.